tools/nl/embeddings/testdata/customdc/expected/custom_embeddings.yaml

```diff
@@ -0,0 +1 @@
+custom_ft: /fake/path/to/custom_embeddings.csv
```

Comment:
nit: use folder "custom_dc" to be consistent with other folders.

Reply:
Done.